pkg/cli/localkubernetes/configure.go

```diff
@@ -190,12 +213,32 @@ func buildDockerCommand(physicalRawConfig clientcmdapi.Config, proxyName, vClust
 			"--network=host",
 			proxyImage,
 			"port-forward",
-			"svc/"+vClusterName,
+			"svc/" + vClusterName,
 			"--address=0.0.0.0",
-			strconv.Itoa(localPort)+":443",
+			strconv.Itoa(localPort) + ":443",
 			"--kubeconfig", "/kube-config",
 			"-n", vClusterNamespace,
-		)
+		}
+		if os.Getenv(constants.DebugLogBackgroundProxyEnvVar) == "1" {
+			args = []string{
+				"run",
+				"--rm",
+				"-d",
+				"-v", fmt.Sprintf("%v:%v", kubeConfigPath, "/kube-config"),
+				fmt.Sprintf("--name=%s", proxyName),
+				"--network=host",
+				proxyImage,
+				"port-forward",
+				"-v7",
```

Comment:
Instead of duplicating the arguments, wouldn't it be better to just append the verbosity level to the slice of args?